genie_annotations.json

```diff
@@ -0,0 +1,24 @@
+{
+  "tissueSource": [
+    "Blood",
+    "Solid Tumor"
+  ],
+  "center": [
+    "NKI",
+    "DFCI",
+    "GRCC",
+    "JHU",
+    "MSK",
+    "UHN",
+    "VICC",
+    "MDA"
+  ],
+  "platform": [
```

Comment:
Platform is a key in the project-level dictionaries; values are thus not controlled (currently). We might not want it in minified, since platforms could vary widely across projects using different techologies.